gateway/dns.go

```diff
@@ -4,16 +4,12 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/ipfs/boxo/autoconf"
 	"github.com/libp2p/go-doh-resolver"
 	dns "github.com/miekg/dns"
 	madns "github.com/multiformats/go-multiaddr-dns"
 )
 
-var defaultResolvers = map[string]string{
-	"eth.":    "https://dns.eth.limo/dns-query",
-	"crypto.": "https://resolver.unstoppable.io/dns-query",
```

Comment:
We should flag that this is dead / removed now (it's not in the default autoconf resolver or in the fallbacks)